cds_ils/importer/providers/ebl/ignore_fields.py

```diff
@@ -17,6 +17,7 @@
     "040__e",
     "040__c",
     "040__d",
+    "050_4b",
```

Comment:
What's the criteria behind ignoring some fields?

Reply:
usually a request from the librarian